docs/app-dev/indexing-transactions.md

```diff
@@ -276,3 +276,12 @@ This behavior was fixed with CometBFT 0.34.26+. However, if the data was indexed
 Tendermint Core and not re-indexed, that data will be queried as if all the attributes within a height
 occurred within the same event.
 
+## Event attribute value types
+
+Users can use anything as an event value. However, if the event attribute value is a number, the following needs to be taken into accound:
+
+- Negative numbers will not be properly retrieved when querying the indexer.
+- Event values are converted to big floats( from the `big/math` package). The precision of the floating point number is set to the bit length 
```

Comment:
```suggestion
- Event values are converted to big floats (from the `big/math` package). The precision of the floating point number is set to the bit length so that there is no loss of precision in the conversion.
```